discord/ui/view.py

```diff
@@ -280,11 +339,11 @@ def add_item(self, item: Item) -> None:
         TypeError
             An :class:`Item` was not passed.
         ValueError
-            Maximum number of children has been exceeded (25)
+            Maximum number of children has been exceeded (40)
             or the row the item is trying to be added to is full.
         """
 
-        if len(self.children) > 25:
+        if len(self.children) >= 40:
```

Comment:
```suggestion
        if len(self.children) > 40:
```

Reply:
if there's already 40 items then you can't add more, hence >=